frontend/src/app/main/ui/ds/notifications/context-notification.stories.jsx

```diff
@@ -30,7 +30,7 @@ export default {
     isHtml: false,
     type: "context",
     appearance: "neutral",
-    level: "info",
```

Comment:
Shouldn't be interesting to also add a new Story for the default level, next to the Info, Error, etc. stories below?

Reply:
You are right, otherwise might be confusing